plugins/frustum_culling/src/frustum_culling_conf.hpp

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2024 Robert Bosch GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * \file frustum_culling_conf.hpp
+ */
+
+#pragma once
+
+#include <memory>   // for shared_ptr<>
+#include <random>   // for default_random_engine, normal_distribution<>
+#include <string>   // for string
+#include <utility>  // for move
+
+#include <cloe/component.hpp>          // for Component, ComponentFactory, ...
+#include <cloe/component/frustum.hpp>  // for Frustum
+#include <cloe/core.hpp>               // for Confable, Schema
+#include <cloe/entity.hpp>             // for Entity
+#include <cloe/simulator.hpp>          // for ModelError
+#include <cloe/utility/geometry.hpp>   // for quaternion_from_rpy
+#include <fable/utility/eigen.hpp>     // for to_json
+
+namespace cloe::frustum_culling_plugin {
+
+/**
+ * This class describes the frustum of a sensor.
+ */
+struct MountPose : public fable::Confable {
+  Eigen::Isometry3d pose;
+  double x, y, z;
+  double roll, pitch, yaw;
+
+  void to_json(fable::Json& j) const override {
+    j = fable::Json{
+        {"pose", pose},
+    };
+  }
```

Comment:
Why only output pose?

Reply:
i think the idea by Tobi was to internally work with Isometry3d class, while from the outside (yamls) being able to set the values x,y,z,roll,pitch,yaw.
So writing pose as well as x,y,z,roll,pitch,yaw to json would contain redundant information. 
Not sure if this is a good class design utilizing this "convert()" function and have redundant information in the class.
Im free to alternative ideas here, maybe there is a better mechanism than the current one? E.g. just store the pose somewhere else and not in the config_?
I think initially Tobi used "from_json" or something similar in order to do this the conversion but that did not work - it seemed to be never called and no conversion form xyz rpy to isometry3d happened.

Reply:
Ok, makes sense.